### ISSUE TITLE
Make Settings Menu tear-offable

### ISSUE THREAD

**Issue by macMikey** [gcanyon/navigator#3]
It would be great if I could have the settings menu be its own palette, either by tearing it off or by a separate setting

**Comment by gcanyon** [comment]
Yeah, since I've taken the plunge with the behavior exporter, there's
definitely a preference panel in the future.
